test-cypress/cypress/e2e/2-advanced-examples/spies_stubs_clocks.cy.js

```diff
@@ -0,0 +1,204 @@
+/// <reference types="cypress" />
+
+context('Spies, Stubs, and Clock', () => {
+  it('cy.spy() - wrap a method in a spy', () => {
+    // https://on.cypress.io/spy
+    cy.visit('https://example.cypress.io/commands/spies-stubs-clocks');
+
+    const obj = {
+      foo () {}
+    };
+
+    const spy = cy.spy(obj, 'foo').as('anyArgs');
+
+    obj.foo();
+
+    expect(spy).to.be.called;
```

Comment:
The assertion `expect(spy).to.be.called;` might not work as expected because it should be `expect(spy).to.have.been.called;` to properly check if the spy was called.